### ISSUE TITLE
Set game to load at start

### ISSUE THREAD

**Issue by cwscws** [revvv/gdash-export-CrLi#81]
Would it be possible to add an option in the preferences to set the game that should be loaded at start?

**Comment by revvv** [comment]
Yes, it would be possible. I guess you know, that you can specify the game to load on the command line? But this is not enough?

**Comment by cwscws** [comment]
Just to be able to start the game without any command line option - would be great! :)

Background is that I want to play a specific game until I finished all caves. But I want to play some today and some tomorrow and so on - so I do not have to load that game every time again.

**Comment by revvv** [comment]
Ok, some thoughts:
- Remember last game played
- Specify a specific game to always load
- Load default game
Maybe combo box with these three options.

**Comment by cwscws** [comment]
Great idea!

**Comment by zsommk** [comment]
It's a very good idea for me too.

**Comment by revvv** [comment]
Just had a look:
- Settings does not support file dialogs.
- ~~`gdash.ini` currently does not support string values.~~ (Sorry, was blind.)
The current generic implementation (which is awesome BTW) needs to revised a lot. ☹

**Comment by renyxadarox** [comment]
> Just to be able to start the game without any command line option - would be great! :)
> 
> Background is that I want to play a specific game until I finished all caves. But I want to play some today and some tomorrow and so on - so I do not have to load that game every time again.

Don't close GDash + OS hibernation (laptop) = that's all you need.
